### PR TITLE
Stock correction warehouse location

### PR DESCRIPTION
**Pull request by dineshkumarj86**
## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```

## Checklist
Have you done the following?
```
[X] Exported any new types in index.ts
[X] Merged master
[X] Run `npm run lint`
[ ] Ensure the tests pass
```

## What's new?
Added API For Default Location And Stock Correction

### CODE REVIEW

File: src/Endpoints.ts [hunk 1]
```diff
@@ -21,7 +21,9 @@ const ENDPOINTS  = Object.freeze({
         get: `${WAREHOUSE}/warehouse`,
         goods_note: {
             search: `${WAREHOUSE}/goods-note/goods-out-search`
-        }
+        },
+        stock_correction: `${WAREHOUSE}/warehouse/`,
+        default_location: `${WAREHOUSE}/warehouse/`
```
Comment (carlrc) [inline src/Endpoints.ts]: Should remove final `/` as it's being added in the client

File: src/IBrightpearlClient.ts [hunk 2]
```diff
@@ -746,6 +746,25 @@ interface CustomFieldMetadataInformation {
     options?: object;
 }
 
+interface StockCorrections {
+    corrections: StockCorrection[];
+}
+
+interface StockCorrection {
```
Comment (carlrc) [inline src/IBrightpearlClient.ts]: Should export StockCorrection also

Comment (dineshkumarj86) [reply src/IBrightpearlClient.ts]: Have done that

File: src/Endpoints.ts (hunk 1 above)
Comment (dineshkumarj86) [reply src/Endpoints.ts]: Have done that.

**Pull request merged**